### DISCUSSION TITLE
Move assets to central/CDN location

### DISCUSSION THREAD

**Discussion by dgcollard** [CRUKorg/cruk-react-components#254] [General]
https://ccp-s3.int.cruk.org/fonts/MuseoSansRounded-500.woff

**Comment by Develliot** [comment]
@dgcollard Do we have a similar url for the SU2C heading font?

**Comment by Develliot** [comment]
I'm turning this into a proper discussion so we can figure our a good deployment strategy for this and have this repo manage it's own assets @sdrycroft  @peteainsworth
